### ISSUE TITLE
Export to PDF - Your Tablature is not valid. Please correct it and try again.

### ISSUE THREAD

**Issue by pattrickrice** [kulek1/guitar-tab-maker-issues#16]
**Describe the bug**

Error dialog appears when trying to export to PDF. Error below is found in the console.

```console
index-TEKWnxS3.js:73 Error: Parse error on line 6:
...13/3 13/3 16/4 13/3 .16/4h 13/3 .16/4h 1
-----------------------^
Expecting 'EOF', 'OPTIONS', 'TABSTAVE', 'STAVE', 'VOICE', 'TEXT', 'NOTES', 'SLUR', '=', '[', ']', '|', ':', '(', 'NUMBER', 'h', '-', 's', 't', 'T', 'b', 'p', '^', '$', '!', '#', 'ABC', got '.'
    at Parser.parseError (vextab.js:47174:21)
    at Parser.parse (vextab.js:47241:22)
    at vextab../src/vextab.jison.exports.parse (vextab.js:47773:51)
    at VexTab.parse (vextab.js:46690:74)
    at oO (index-TEKWnxS3.js:299:103952)
    at fO (index-TEKWnxS3.js:305:52)
    at m (index-TEKWnxS3.js:309:15476)
    at index-TEKWnxS3.js:70:6984
    at Object.wv (index-TEKWnxS3.js:99:9855)
    at yv (index-TEKWnxS3.js:99:10009)
    at Sv (index-TEKWnxS3.js:99:10066)
    at W0 (index-TEKWnxS3.js:99:31446)
    at KQ (index-TEKWnxS3.js:99:31863)
    at index-TEKWnxS3.js:99:36776
    at oV (index-TEKWnxS3.js:102:36935)
    at uQ (index-TEKWnxS3.js:99:8991)
    at cg (index-TEKWnxS3.js:99:33150)
    at Rp (index-TEKWnxS3.js:99:17360)
    at Wv (index-TEKWnxS3.js:99:17144)
    at HTMLDivElement.r (index-TEKWnxS3.js:358:1426)
```

**To Reproduce**
I've included the export of the tab to TEXT which it exports successfully.
[project37.txt](https://github.com/kulek1/guitar-tab-maker-issues/files/13933042/project37.txt)

**Expected behavior**

**Screenshots**

<img width="541" alt="Screenshot 2024-01-14 at 12 28 09 PM" src="https://github.com/kulek1/guitar-tab-maker-issues/assets/20763171/0c4b2641-84be-4160-9012-b3249278344d">

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser Chrome
 - Version 120.0.6099.199

**Additional context**

I actually don't need to export to PDF. I just need a way to print the tabs.

**Comment by kulek1** [comment]
@pattrickrice thanks for the report!
Given what you have provided, it seems to me that the second tablature you have created might be incorrect:
```
E|--------------------------------------------------------------------------------------
B|--------------------------------------------------------------------------------------
G|-16-16-13-13-16-16-13-13-16-16-13-13-16-16-13-13-----13-h--13-h--13-h--13-h-13----13--
D|-------------------------------------------------16----16----16----16----16----16-----
A|--------------------------------------------------------------------------------------
E|--------------------------------------------------------------------------------------
```
specifically this part:
<img width="210" alt="Screenshot 2024-01-14 at 22 04 36" src="https://github.com/kulek1/guitar-tab-maker-issues/assets/7605514/53036d1d-30ad-45df-a3e0-cfaacde133e5">

Can you attach a screenshot of that tablature to make sure I'm not missing anything?

**Comment by pattrickrice** [comment]
Here is a screenshot of that section. 

<img width="515" alt="Screenshot 2024-01-14 at 12 27 59 PM" src="https://github.com/kulek1/guitar-tab-maker-issues/assets/20763171/0cff3149-72c3-4bdc-9e78-9212510cd6df">

I think I would be surprised that whether the section is "valid" or not would cause an error, especially since the export to text works fine? Validating the tab could make sense, but maybe highlighting the erroneous section would be useful.

**Comment by kulek1** [comment]
@pattrickrice thanks! Correct me if I'm wrong, but hammer-on from 13 to 13 doesn't make sense and that's why the PDF doesn't work here. I'll think about adding a validation for invalid tabs.

Although your problem doesn't suggest a bug in the application, I have released a new version `v3.5.0` which supports the "print" function from your browser. I hope you'll now be able to print your tabs :)

**Comment by pattrickrice** [comment]
Wow super appreciate the fast turn around on that! Yeah i was misusing the `h`, but thanks for considering the suggestion!
